Calibration/TkAlCaRecoProducers/python/ALCARECOPromptCalibProdSiStripHitEfficiency_cff.py

```diff
@@ -0,0 +1,87 @@
+import FWCore.ParameterSet.Config as cms
+
+# ------------------------------------------------------------------------------
+# configure a filter to run only on the events selected by TkAlMinBias AlcaReco
+import copy
+from HLTrigger.HLTfilters.hltHighLevel_cfi import *
+ALCARECOCalMinBiasFilterForSiStripHitEff = copy.deepcopy(hltHighLevel)
+ALCARECOCalMinBiasFilterForSiStripHitEff.HLTPaths = ['pathALCARECOSiStripCalMinBias']
+ALCARECOCalMinBiasFilterForSiStripHitEff.throw = True ## dont throw on unknown path names
+ALCARECOCalMinBiasFilterForSiStripHitEff.TriggerResultsTag = cms.InputTag("TriggerResults","","RECO")
+
+# ------------------------------------------------------------------------------
+# This is the sequence for track refitting of the track saved by SiStripCalMinBias
+# to have access to transient objects produced during RECO step and not saved
+from Alignment.CommonAlignmentProducer.AlignmentTrackSelector_cfi import *
+ALCARECOMonitoringTracks = AlignmentTrackSelector.clone(
+    #    src = 'generalTracks',
+    src = 'ALCARECOSiStripCalMinBias',
+    filter = True,
+    applyBasicCuts = True,
+    ptMin = 0.8,
+    nHitMin = 6,
+    chi2nMax = 10.)
+
+# FIXME: the beam-spot should be kept in the AlCaReco (if not already there) and dropped from here
+from RecoVertex.BeamSpotProducer.BeamSpot_cff import *
+from RecoTracker.IterativeTracking.InitialStep_cff import *
+from RecoTracker.Configuration.RecoTrackerP5_cff import *
+from RecoTracker.TrackProducer.TrackRefitter_cfi import *
+from DQM.SiStripCommon.TkHistoMap_cff import *
+from RecoTracker.MeasurementDet.MeasurementTrackerEventProducer_cfi import *
+
+ALCARECOMonitoringTracksRefit = TrackRefitter.clone(src = cms.InputTag("ALCARECOMonitoringTracks"),
+                                                     NavigationSchool = cms.string("")
+                                                     )
+
+# ------------------------------------------------------------------------------
+# refit and BS can be dropped if done together with RECO.
+# track filter can be moved in acalreco if no otehr users
+ALCARECOTrackFilterRefit = cms.Sequence(ALCARECOMonitoringTracks +
+                                        MeasurementTrackerEvent +
+                                        offlineBeamSpot +
+                                        ALCARECOMonitoringTracksRefit)
+
+# ------------------------------------------------------------------------------
+# This is the module actually doing the calibration
+from CalibTracker.SiStripHitEfficiency.siStripHitEfficiencyWorker_cfi import siStripHitEfficiencyWorker
+ALCARECOSiStripHitEff =  siStripHitEfficiencyWorker.clone(
+    lumiScalers=cms.InputTag("scalersRawToDigi"),
```

Comment:
Please remove type specification for the cloned parameters, which is a safer way to overwrite them

Reply:
done at https://github.com/cms-sw/cmssw/pull/37708/commits/5e79159a730dd71c6faf73ff474272440903dace